dm/dm/worker/server.go

```diff
@@ -111,6 +111,7 @@ func (s *Server) Start() error {
 		DialKeepAliveTime:    keepaliveTime,
 		DialKeepAliveTimeout: keepaliveTimeout,
 		TLS:                  tls.TLSConfig(),
+		AutoSyncInterval:     syncMasterEndpointsTime,
```

Comment:
this can start a thread to sync endpoint in etcdclient see more here https://pkg.go.dev/go.etcd.io/etcd/clientv3#Config